adafruit_bitbangio.py

```diff
@@ -304,15 +312,15 @@ def __init__(self, clock, MOSI=None, MISO=None):
             self._miso = DigitalInOut(MISO)
             self._miso.switch_to_input()
 
-    def deinit(self):
+    def deinit(self) -> None:
         """Free any hardware used by the object."""
         self._sclk.deinit()
         if self._miso:
             self._miso.deinit()
         if self._mosi:
             self._mosi.deinit()
 
-    def configure(self, *, baudrate=100000, polarity=0, phase=0, bits=8):
+    def configure(self, *, baudrate: int = 100000, polarity: Literal[0,1] = 0, phase: Literal[0,1] = 0, bits: int = 8) -> None:
```

Comment:
Nice use of `Literal`!  Mind separating the `0` and `1` with spaces (`Literal[0, 1]`)?